src/Chr.Avro/Chr.Avro.csproj

```diff
@@ -6,6 +6,7 @@
     <Description>Avro schema models</Description>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <LangVersion>7.2</LangVersion>
```

Comment:
any particular reason for this change?

Reply:
Oh Visual Studio 2017 was bugging about  it... you could remove it I guess... now moved to VS 2019